packages/mds-agency/api.ts

```diff
@@ -15,48 +15,25 @@
  */
 
 import express from 'express'
-import urls from 'url'
 
 import log from '@mds-core/mds-logger'
-import db from '@mds-core/mds-db'
-import cache from '@mds-core/mds-cache'
-import stream from '@mds-core/mds-stream'
-import { providerName, isProviderId } from '@mds-core/mds-providers'
-import areas from 'ladot-service-areas'
-import {
-  UUID,
-  Recorded,
-  Device,
-  VehicleEvent,
-  Telemetry,
-  ErrorObject,
-  Timestamp,
-  DeviceID,
-  isEnum,
-  VEHICLE_EVENTS,
-  VEHICLE_TYPES,
-  VEHICLE_STATUSES,
-  VEHICLE_REASONS,
-  PROPULSION_TYPES,
-  EVENT_STATUS_MAP,
-  VEHICLE_STATUS,
-  VEHICLE_EVENT,
-  BoundingBox,
-  VEHICLE_REASON
-} from '@mds-core/mds-types'
-import {
-  isUUID,
-  isPct,
-  isTimestamp,
-  isFloat,
-  pointInShape,
-  now,
-  pathsFor,
-  ServerError,
-  isInsideBoundingBox
-} from '@mds-core/mds-utils'
+import { isProviderId } from '@mds-core/mds-providers'
+import { isUUID, pathsFor } from '@mds-core/mds-utils'
 import { AgencyApiRequest, AgencyApiResponse } from '@mds-core/mds-agency/types'
 import { checkScope } from '@mds-core/mds-api-server'
+import {
+  getAllServiceAreas,
```

Comment:
Did your fork fall behind perhaps? I see these changes in `develop` already.

Reply:
sure thing!